src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -638,54 +639,47 @@ This resets the cluster to its initial state.`,
 	subcommands = append(subcommands, cmdutil.CreateAlias(portForward, "port-forward"))
 
 	var install bool
-	var path string
-	completion := &cobra.Command{
+	var installPathBash string
+	completionBash := &cobra.Command{
 		Short: "Print or install the bash completion code.",
-		Long:  "Print or install the bash completion code. This should be placed as the file `pachctl` in the bash completion directory (by default this is `/etc/bash_completion.d`. If bash-completion was installed via homebrew, this would be `$(brew --prefix)/etc/bash_completion.d`.)",
+		Long:  "Print or install the bash completion code.",
 		Run: cmdutil.RunFixedArgs(0, func(args []string) (retErr error) {
-			var dest io.Writer
-
 			if install {
-				f, err := os.Create(path)
-
-				if err != nil {
-					if os.IsPermission(err) {
-						fmt.Fprintf(os.Stderr, "Permission error installing completions, rerun this command with sudo.\n")
-					}
-					return err
+				if path.Base(installPathBash) != "pachctl" {
```

Comment:
This branch is screwing me up - if I do `--path ~/.bash_completions.pachctl`, it tries to write to `~/.bash_completions.pachctl/pachctl`, which obviously doesn't work.  What is this branch supposed to handle?

Reply:
Is that a valid path for a bash completion? AFAIK, bash always uses `<completion directory>/<name of executable>`.

Reply:
I don't know if there are that many conventions for completion scripts - I found one place that said to just source the scripts in your .bashrc.